src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -137,14 +160,13 @@ This adapter does not support the `image.domains` and `image.remotePatterns` con
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 155 to 157]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Not sure if this should be a warning?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/withastro/docs/5978?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Good call. I can see that!